runtime/onert/core/include/ir/Graph.h

```diff
@@ -119,6 +119,7 @@ class Graph : public IGraph
   // Topological sort
 public:
   std::vector<ir::OperationIndex> topolSortOperations() const;
+  std::vector<ir::OperationIndex> btopolSortOperations() const;
```

Comment:
`Graph` is used for `ir::operation`. `btopolSortOperations` is only for training. I don't want to add this in `ir::Graph`.

Reply:
@jyoungyun How about moving it to `TrainableGraph`?

Reply:
Okay, it's a good idea. However even though this function is moved to the `TrainableGraph`, the Linear still needs the `blinearize()` function. I'm worried about it...

Reply:
@ragmani I fixed this PR. I do not make a wrapper function for `btopolSortOperations()` in `Linear` class.

Reply:
Hm... I think it's okay to add it to there.
If we have to put it in a train directory, how about introducing a new `Linear` class for training?